src/test/Slasher.t.sol

```diff
@@ -255,14 +287,26 @@ contract SlasherTests is EigenLayerTestHelper {
 
         //can opt in after registered as operator
         cheats.startPrank(_operator);
-        delegation.registerAsOperator(delegationTerms);
+        IDelegationManager.OperatorDetails memory operatorDetails = IDelegationManager.OperatorDetails({
+            earningsReceiver: _operator,
+            delegationApprover: address(0),
+            stakerOptOutWindowBlocks: 0
+        });
+        string memory emptyStringForMetadataURI;
+        delegation.registerAsOperator(operatorDetails, emptyStringForMetadataURI);
         slasher.optIntoSlashing(_slasher);
         cheats.stopPrank();
     }
 
     function testFreezeOperator() public {
         cheats.prank(operator);
-        delegation.registerAsOperator(delegationTerms);
+        IDelegationManager.OperatorDetails memory operatorDetails = IDelegationManager.OperatorDetails({
```

Comment:
make this an internal function?

Reply:
these are inside of the "Slasher" tests. I didn't want to mess with them since some blocks have additional logic (e.g. 'opting into slashing') but agree it's a good candidate for de-duplicating